pages/api/auth/login.tsx

```diff
@@ -0,0 +1,57 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { SignJWT, importJWK } from 'jose';
+import bcrypt from 'bcryptjs';
```

Comment:
Apparently there's an error with bcrypt https://github.com/kelektiv/node.bcrypt.js/issues/979

Trying this alternative, but this might actually be causing more problems. I'll keep an eye on it.